### PR TITLE
Support Distinct function in query

### PR DESCRIPTION
**Pull request by bege13mot**
This PR introduces support for the Distinct function in the query string. This hopefully solves prest/prest#221.

### CODE REVIEW

**Review by felipeweb — CHANGES_REQUESTED** [review]
i think the better way to implement this is create `?_distinct=true`

**Comment by bege13mot** [comment]
Sorry, @felipeweb I'm new in golang and in this project. Could you please provide me an example? It isn't clear for me now.

**Comment by felipeweb** [comment]
@bege13mot  a new param in query string called distinct

**Comment by lerrua** [comment]
@bege13mot great PR, but as @felipeweb wrote above it's better change distinct to `_distinct` to follow others query params. You can see our examples here: https://postgres.rest/query-statements/

File: postgres/postgres.go [hunk 1]
```diff
@@ -97,13 +97,15 @@ func Prepare(db *sqlx.DB, SQL string) (stmt *sql.Stmt, err error) {
 // chkInvalidIdentifier return true if identifier is invalid
 func chkInvalidIdentifier(identifer ...string) bool {
 	for _, ival := range identifer {
+		lower := strings.ToLower(ival)
 		if ival == "" || len(ival) > 63 || unicode.IsDigit([]rune(ival)[0]) {
 			return true
-		}
+		}
```
Comment (avelino) [inline postgres/postgres.go:103]: run gofmt pls

**Review by avelino — CHANGES_REQUESTED** [review]

**Comment by bege13mot** [comment]
Thank you @felipeweb and @lerrua for the explanations. Hope my new pull request will be better.

**Pull request closed without merging**